script/Ciel/Day 5/05_01_CIEL05_1C.txt

```diff
@@ -186,6 +186,7 @@ I slam my head against the iron bars in anger.
 [sha:61775d325b1cd96cf258c64d9765ad04db73b3c1]{
 -- Page 100, Offset 11140.
 -- 人として正常な判断を、過剰な痛みで封殺する。
+// force those thoughts? is this supposed to be force out or drive out?
```

Comment:
its separated by em dashes. The sentence is "I force those thoughts from my mind". No change necessary